gear-programs/bridging-payment/app/src/services/mod.rs

```diff
@@ -95,35 +95,23 @@ where
     }
 
     fn state(&self) -> &State {
-        unsafe {
-            STATE
-                .as_ref()
-                .expect("BridgingPayment::seed() should be called")
-        }
+        #[allow(clippy::deref_addrof)]
+        unsafe { (*&raw const STATE).as_ref() }.expect("BridgingPayment::seed() should be called")
```

Comment:
update gstd if possible 
https://docs.rs/gstd/latest/gstd/macro.static_mut.html

Reply:
Thanks I am aware of that macro however we strictly bound to sails so cannot use newer gstd with an old sails.

Created corresponding issue - #272